### PR TITLE
Implementation of grayscale

### PR DESCRIPTION
**Pull request by Enrique-C**

### CODE REVIEW

File: src/main/java/com/jalasoft/jfc/model/image/ImageParam.java [hunk 1]
```diff
@@ -26,6 +26,9 @@
     // Contains a boolean value for building a Thumbnail.
     private boolean isThumbnail;
 
+    // Contains a boolean value for building a Thumbnail.
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/image/ImageParam.java:29]: This variable is "isGrayscale" not thumbnail

File: src/main/java/com/jalasoft/jfc/model/strategy/CommandImageGrayscale.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    /**
+     * Allows to instantiate this class.
+     * @param commandValue receives a boolean value.
+     */
+    public CommandImageGrayscale(boolean commandValue) {
+        this.commandValue = commandValue;
+    }
+
+    /**
+     * This method builds a command.
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/strategy/CommandImageGrayscale.java]: Delete this method.

File: src/main/java/com/jalasoft/jfc/model/strategy/CommandImageGrayscale.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+     * @return command concatenated.
+     * @throws CommandValueException generates a error message.
+     */
+    @Override
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/strategy/CommandImageGrayscale.java:40]: Put a new line

File: src/main/java/com/jalasoft/jfc/model/strategy/CommandImageGrayscale.java (hunk 2 above)
Comment (Enrique-C) [reply src/main/java/com/jalasoft/jfc/model/strategy/CommandImageGrayscale.java]: What is the reason ?

File: src/main/java/com/jalasoft/jfc/model/strategy/CommandImageGrayscale.java (hunk 3 above)
Comment (Enrique-C) [reply src/main/java/com/jalasoft/jfc/model/strategy/CommandImageGrayscale.java:40]: What is the reason to do that?

File: src/main/java/com/jalasoft/jfc/model/image/ImageParam.java (hunk 1 above)
Comment (Enrique-C) [reply src/main/java/com/jalasoft/jfc/model/image/ImageParam.java:29]: It is done.

**Review by oscarlopez8 — APPROVED** [review]

**Pull request merged**